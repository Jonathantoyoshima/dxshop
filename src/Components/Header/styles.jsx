import styled from 'styled-components';

export const Root = styled.div`
	width: 100%;
	padding: 8px;
	text-align: center;
	display: flex;
	justify-content: center;
	align-items: center;
	position: relative;
	color: ${(props) => props.theme.color.primary.dark};
	background-color: ${(props) =>
		props.theme.color.primary.light};
`;
