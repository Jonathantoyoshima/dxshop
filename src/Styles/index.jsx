import styled from 'styled-components';

export const theme = {
	typography: {
		normal: `font-family: Arial, sans; letter-spacing: 0.1rem;`,
	},
	color: {
		basic: {
			black: '#0c0c0c',
			white: '#fff',
		},
		primary: {
			main: '#4f6f8c',
			light: '#b4c4d9',
			dark: '#254559',
		},
		secondary: {
			light: '#4D4D99',
			dark: '#011c26',
		},
	},
};

export const General = styled.div`
	${(props) => props.theme.typography.normal};
`;

export const Container = styled.div`
	position: relative;
	width: 100%;
	max-width: 1080px;
	padding: 0 16px;
`;

export const Wrapper = styled.div`
	display: flex;
	flex-direction: column;
	align-items: center;
	justify-content: center;
	color: ${(props) => props.theme.color.basic.white};
	background-color: ${(props) =>
		props.theme.color.secondary.dark};
`;

export const Button = styled.button`
	padding: 10px 30px;
	text-transform: uppercase;
	letter-spacing: 0.3rem;
	border-radius: 10px;
	background-color: ${(props) =>
		props.theme.color.primary.dark};
	border: 1px solid
		${(props) => props.theme.color.basic.white};
	color: ${(props) => props.theme.color.basic.white};
	cursor: pointer;
	transition: all 1s ease;
	:hover {
		background-color: ${(props) =>
			props.theme.color.primary.light};
		border: 1px solid
			${(props) => props.theme.color.primary.dark};
		color: ${(props) => props.theme.color.primary.dark};
	}
`;

export const Input = styled.input`
	padding: 8px;
	width: 100%;
	box-shadow: inset -5px -5px 20px 0px rgba(0, 0, 0, 0.5);
`;
