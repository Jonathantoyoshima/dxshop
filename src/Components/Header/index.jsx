import { Root } from './styles';

const Header = ({ children }) => {
  return <Root>{children}</Root>;
};

export default Header;
