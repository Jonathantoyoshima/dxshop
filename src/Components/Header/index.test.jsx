import React from 'react';
import renderer from 'react-test-renderer';
import { ThemeProvider } from 'styled-components';
import { theme } from 'Styles';
import Header from './index';

test('Render Build Header Snapshot', () => {
	const component = renderer.create(
		<ThemeProvider theme={theme}>
			<Header />
		</ThemeProvider>
	);
	let tree = component.toJSON();
	expect(tree).toMatchSnapshot();
});
