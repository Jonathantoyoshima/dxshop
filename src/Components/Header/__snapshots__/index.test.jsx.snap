// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render Build Header Snapshot 1`] = `
<div
  className="sc-jSgupP juqtVP"
/>
`;
